server/models/visits.js

```diff
@@ -83,6 +83,7 @@ var visits = {
                 fields: [
                   'extractedTitle',
                   'extractedContent',
+                  'extractedDescription',
```

Comment:
Good points. Let's do a follow up PR for these. @pdehaan will you create an issue?